command.go

```diff
@@ -1183,6 +1183,10 @@ func (cmd *CustomCmd) String() string {
 	return cmdString(cmd, cmd.customReader)
 }
 
+func (cmd *CustomCmd) Result() error {
```

Comment:
It was a drawback in previous variants of this function
Error should be read and reported